client/src/components/Home/index.jsx

```diff
@@ -82,10 +81,10 @@ class Home extends PureComponent {
       isLoading: true
     });
 
+    toastr.remove();
     if (this.state.password === this.state.password2) {
```

Comment:
Use a descriptive name for your variables e.g. password2 could be confirmPassword